bundle/types.go

```diff
@@ -152,6 +152,19 @@ type Spec struct {
 	Async       string                 `json:"async"`
 	Plans       []Plan                 `json:"plans"`
 	Alpha       map[string]interface{} `json:"alpha,omitempty"`
+	Delete      bool                   `json:"delete"`
+}
+
+// MarkDelete - marks the spec for deletion, which indicates that the spec
+// will eventually be deleted from the datastore
+func (s *Spec) MarkDelete() {
```

Comment:
I personally do not prefer these helper methods. Delete being a bool I think the Caller could easily update it IMO.

Reply:
+1. If this were likely to change in the future it would warrant a fn, but I don't think this is worth it.